src/vs/workbench/contrib/chat/common/chatProgressTypes/chatToolInvocation.ts

```diff
@@ -84,6 +88,14 @@ export class ChatToolInvocation implements IChatToolInvocation {
 		return this._confirmationMessages;
 	}
 
+	public acceptProgress(step: IProgressStep) {
```

Comment:
Is the `increment`-property actually used/displayed anywhere?

Reply:
Not yet, but MCP supports this and I want to have an indicator for it in the future. Maybe a spinner like the Windows one where it gradually fills up as progress is incremented.

Reply:
Cool. We also have the `ProgressBar` component which can render this